### PR TITLE
Feature EPL-1727: First model of the Breadcrums component

### PR DESCRIPTION
**Pull request by Sankku**
First model of the breadcrum component

image

### CODE REVIEW

**Review by tomyrodeghiero — APPROVED** [review]

**Review by sjcuello — APPROVED** [review]

File: packages/ComponentLibrary/src/components/Breadcrums/index.tsx [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+        {renderBreadcrumbItems()}
+      </Breadcrumbs>
+      <Menu
+        anchorEl={anchorEl}
+        open={Boolean(anchorEl)}
+        onClose={handleActionMenuClose}
+        slotProps={{
+          paper: { sx: sx.menu, elevation: 3 },
+        }}
+        MenuListProps={{ sx: menuStyle }}>
```
Comment (LuuchoRocha) [inline packages/ComponentLibrary/src/components/Breadcrums/index.tsx:198]: move out this into a constant

File: packages/ComponentLibrary/src/components/Breadcrums/index.tsx [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+          <MenuItem key={action.id} onClick={() => {}} sx={sx.menuItem}>
+            <Box sx={sx.iconBox}>
+              {action.icon}
+              <span>{action.label}</span>
+            </Box>
+            {action.toggle && (
+              <Box sx={sx.toggleContainer}>
+                <ToggleChip
+                  isActive={toggleStates[action.id] ?? false}
+                  onToggle={() => handleToggle(action.id)}
```
Comment (LuuchoRocha) [inline packages/ComponentLibrary/src/components/Breadcrums/index.tsx:209]: note to future ourselves: we need to develop some helper to create this handlers inside map loops and avoid the usage of arrow functions

**Review by LuuchoRocha — CHANGES_REQUESTED** [review]
Excellent job!
The only missing thing is to address the callbacks issues

**Review by tomyrodeghiero — DISMISSED** [review]
I agree with @LuuchoRocha's comments

**Pull request merged**